applications/asset_tracker_v2/tests/nrf_cloud_codec_mocked_cjson/src/cJSON_to_mock.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "cJSON.h"
+
+/* This file is needed because cmock's generator cannot parse the original header. */
```

Comment:
cJSON has this very playful setup of changing the type specifiers of functions with a macro, e.g.: `CJSON_PUBLIC(const char*) cJSON_Version(void);`

CMock will not recognize these declarations as functions since it skips preprocessing.